docs/src/pages/start-here.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Start Here
+---
+
+# Start here
```

Comment:
Are we supposed to do this in addition to the title above? I don't see that in `about-us.md`.

Reply:
It seems that spec and user-guide do it, and I'm the one who forgot it on about-us.  It might just set the page title for purposes of bookmarks and stuff. Will fix.

Reply:
wait, no, user-guide is the one that doesn't have it. not fixing it here & now.